packages/sampler/stories/qa-stories/Scroller.js

```diff
@@ -28,6 +29,7 @@ storiesOf('Scroller', module)
 		'List of things',
 		() => (
 			<Scroller
+				data-spotlight-container-disabled={boolean('data-spotlight-container-disabled', ScrollableBase, false)}
```

Comment:
Do we need `ScrollableBase` instead of `Scroller` here?

Reply:
No. I missed it. Thanks.